ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -346,6 +349,10 @@ public class RNTrackPlayer: RCTEventEmitter, AudioSessionControllerDelegate {
             interval: ((options["progressUpdateEventInterval"] as? NSNumber) ?? 0).doubleValue
         )
 
+        updateCategory(config: options)
+
+        configureAudioSession()
```

Comment:
@dcvz Can I call this here safely? I realize that reinitialization is something that should be avoided based on this code inside `setupPlayer`:

```
            reject("player_already_initialized", "The player has already been initialized via setupPlayer.", nil)
```

Reply:
I think you can update this safely since the phone switches nicely between the two speakers when updating the iosCategory and iosCategoryMode properties.